tailwind.config.cjs

```diff
@@ -2,7 +2,24 @@
 module.exports = {
   content: ["*.{html,ts}"],
   theme: {
-    extend: {},
+    extend: {
+      height: {
+        8: "2rem",
+        12: "3rem",
```

Comment:
setting the custom field to something that conflicts with tailwind will cause issues.

8 is already 2rem in tailwind and the same with 12 which is 3rem.

Reply:
Had problems picking up the tailwind style, decided to add custom fields. but fixed now thanks for the comment.